python/paddle/fluid/tests/unittests/test_instag_op.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This is unit test of Test filter_instag Op."""
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+import paddle.fluid as fluid
+import paddle.fluid.core as core
+import paddle.fluid.layers as layers
+from op_test import OpTest
+import random
+from decorator_helper import prog_scope
+"""This is Test Case 0"""
+
+
+class TestFilterInstagOp(OpTest):
+    def setUp(self):
+        self.op_type = 'filter_instag'
+        x1 = np.zeros((36, 4), dtype=np.float64)
```

Comment:
x1, x2, x3 is not appropriate for test case, maybe you should give a more meaningful name to x1/x2/x3

Reply:
thank you, I've done it.